src/Clerk.ts

```diff
@@ -133,7 +133,22 @@ export default class Clerk {
 
   // Middlewares
 
-  expressMiddleware({ onError }: MiddlewareOptions = {}) {
+  // defaultOnError doesn't do anything
+  // return an Error with statusCode if you want to halt the request chain
+  defaultOnError(error: Error) {
+    console.error(error.message);
+  }
+
+  // strictOnError returns an error with statusCode 401 so that express will halt the request chain
+  strictOnError(error: Error) {
+    console.error(error.message);
+    const err = new Error("Unauthenticated Access");
+    // @ts-ignore
```

Comment:
❓ Can we avoid this @ts-ignore by defining the error type?

Reply:
Will do